packages/shopify-codemod/transforms/conditional-assign-to-if-statement.js

```diff
@@ -0,0 +1,62 @@
+function isConditionalAssignment(node) {
+  const isUsingOr = (node.operator === '||');
+  const isLeftCheckingVariable = (node.left.type === 'MemberExpression') || (node.left.type === 'Identifier');
+  const isRightAssignment = (node.right.type === 'AssignmentExpression');
+
+  return isUsingOr && isLeftCheckingVariable && isRightAssignment;
+}
+
+function isMemberEqual(aNode, bNode) {
+  return (
+    aNode.object.type === bNode.object.type &&
```

Comment:
I feel like there should be an easier way of finding out if two nodes are equivalent but I couldn't find it :S